source/slang/ir-bind-existentials.cpp

```diff
@@ -342,19 +342,60 @@ struct BindExistentialSlots
         }
         else
         {
-            // TODO: We eventually need to handle cases where there
-            // are:
+            UInt slotOperandCount = slotCount*2;
```

Comment:
Presumably no longer needs SLANG_UNUSED(slotCount);

Why * 2? Interface, Witness table? Interface, Concrete type?

Reply:
The `* 2` is because each slot needs a type operand and a witness-table operand. I'm not super happy with how the code looks when we have to manipulate instructions with "interleaved" operands like this. The main alternative would be to have a specific `IRInst` case that represents a pair of values, and then have things like this have operands that are pairs...